lib/src/models/comment_impl.dart

```diff
@@ -235,7 +235,7 @@ class Comment extends CommentRef
   int get downvotes => data['downs'];
 
   /// Has this [Comment] been edited.
-  bool get edited => data['edited'];
+  bool get edited => (data['edited'] is double);
```

Comment:
Sorry, I totally meant to document this.

This field is either set to `false` or a UTC timestamp depending on whether or not the `Comment` has been edited. If `data['edited'] is double == true`, then the `Comment` has been edited. Otherwise, `data['edited']` should be set to `false` anyway.